app/src/app/components/stix/attackid-property/attackid-edit/attackid-edit.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="attackid-edit">
-    <mat-form-field class="attackid" appearance="outline">
+    <mat-form-field class="attackid" appearance="outline" [matTooltip]="showHint ? 'ID prefix appended is based on your organization\'s namespace settings' : null">
         <mat-label>ID</mat-label>
-        <input matInput [ngModel]="config.object['attackID']" (keyup)="handleKeyup($event.target.value)">
+        <input matInput [(ngModel)]="config.object['attackID']" (blur)="prependPrefix()">
         <button *ngIf="['group', 'mitigation', 'software', 'tactic', 'technique', 'data-source'].includes(config.object['attackType'])" mat-icon-button matSuffix (click)="handleGenerateClick()" matTooltip="Generate a unique ID, with your organization's namespace settings." aria-label="Generate ID button">
             <mat-icon>{{'autorenew'}}</mat-icon>
         </button>
-        <mat-hint *ngIf="config.object['attackType'] === 'matrix' && showHint">ID prefix appended is based on your organization's namespace settings</mat-hint>
+        <mat-hint></mat-hint>
```

Comment:
This looks like it's dead code. There's some code in `prependPrefix()` that flips a `showHint` conditional, but looks like it may have been removed. If this is dead, the hint-related code should be removed, otherwise I think the `matHint` is missing a conditional & hint text.